test/integration/targets/virt_net/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- include_vars: '{{ item }}'
+  with_first_found:
+    - "{{ ansible_distribution }}-{{ ansible_distribution_version}}.yml"
+    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version}}.yml"
+    - "{{ ansible_distribution }}.yml"
+    - "default.yml"
+
+- block:
+    - include_tasks: 'prepare.yml'
+    - name: Define the foobar network
+      virt_net:
+        command: define
+        name: foobar
+        xml: '{{ lookup("file", "foobar.xml") }}'
+    
+    - name: Define the foobar network (again)
+      virt_net:
+        command: define
+        name: foobar
+        xml: '{{ lookup("file", "foobar.xml") }}'
+      register: second_virt_net_define
+    
+    - name: Ensure the second call returns "unchanged"
+      assert:
+        that:
+          - "second_virt_net_define is not changed"
+    
+    - name: Start the default network
+      virt_net:
+        uri: qemu:///system
+        command: start
+        name: foobar
+    
+    - name: Start the default network (again)
+      virt_net:
+        uri: qemu:///system
+        command: start
+        name: foobar
+      register: second_virt_net_start
+    
+    - name: Ensure the second call returns "unchanged"
+      assert:
+        that:
+          - "second_virt_net_start is not changed"
```

Comment:
You can reduce test output by combining the two asserts.

Reply:
ok